flocker/node/_container.py

```diff
@@ -714,4 +710,4 @@ def calculate_changes(self, desired_configuration, current_cluster_state,
             phases.append(in_parallel(changes=start_restart))
 
         return sequentially(changes=phases,
-                            sleep_when_empty=timedelta(seconds=1))
```

Comment:
@itamarst Suggested this in https://clusterhq.atlassian.net/browse/FLOC-3374 along with some logging reduction.